src/main/java/org/reactome/release/resourcechecker/FileResourceChecker.java

```diff
@@ -0,0 +1,149 @@
+package org.reactome.release.resourcechecker;
+
+import com.google.gson.JsonObject;
+
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.Arrays;
+
+public interface FileResourceChecker extends ResourceChecker {
+	void saveFileContents(Path fileDestination) throws IOException;
+
+	long getFileSize();
+
+	/**
+	 * Returns true if the current file size has not dropped more than an acceptable percentage when compared with
+	 * the previous file size.
+	 * @param previousFileSize Last known acceptable file size in bytes
+	 * @param acceptablePercentageDrop Percentage of the drop in file size which is acceptable between the current and
+	 * previous file size
+	 * @return True if the file size has not dropped more than the acceptable percentage; False otherwise
+	 * @see #isFileSizeAcceptable(long)
+	 */
+	default boolean isFileSizeAcceptable(long previousFileSize, double acceptablePercentageDrop) {
+		long differenceInFileSize = getFileSize() - previousFileSize;
+		double percentChangeInFileSize = differenceInFileSize * 100.0d / previousFileSize;
```

Comment:
Does it ever happen where a resource is suddenly much _larger_ than it used to be? I think that is covered by this code but I'm not sure... Maybe change the name of `acceptablePercentageDrop` to `acceptablePercentageChange`?

Reply:
A file increase of any amount is currently acceptable, but you're right - I want to implement something to alert us to a suspicious jump in size.  I'm just thinking of what the threshold would/should be.

Reply:
Created issue #2